.travis.yml

```diff
@@ -14,9 +14,20 @@ env:
 
 before_install:
   # Docker-compose installation
-  - curl -L https://github.com/docker/compose/releases/download/1.8.1/docker-compose-`uname -s`-`uname -m` > docker-compose
-  - chmod +x docker-compose
-  - sudo mv docker-compose /usr/local/bin
+  - sudo apt-get update
+  - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
```

Comment:
@ruflin see the comment below: I get the previous code for docker-compose install ***back***